src-tauri/Cargo.toml

```diff
@@ -43,7 +43,7 @@ sysinfo = "0.26.2"
 # HTTP requests
 reqwest = { version = "0.11", features = ["blocking"] }
 # Persistent store for settings
-tauri-plugin-store = { git = "https://github.com/tauri-apps/tauri-plugin-store", rev = "9bd993aa67766596638bbfd91e79a1bf8f632014" }
+tauri-plugin-store = { git = "https://github.com/tauri-apps/plugins-workspace", branch = "dev" }
```

Comment:
Could you point to a commit instead of a branch?

Reply:
Shouldn't really be necessary but given that the plugin store is still under active development it might be for the best.

Resolved in 22ac2cafd7fb5946a5ba99044572fb3316e59761